stories/InfiniteHits.stories.js

```diff
@@ -124,3 +125,35 @@ storiesOf('ais-infinite-hits', module)
         </div>
       </ais-infinite-hits>`,
   }));
+
+storiesOf('ais-infinite-hits', module)
+  .addDecorator(
+    previewWrapper({
+      insightsClient: (method, payload) =>
+        action(`[InsightsClient] sent ${method} with payload`)(payload),
+    })
+  )
+  .add('with insights', () => ({
+    template: `
+    <div>
+        <ais-configure :clickAnalytics="true" />
+        <ais-infinite-hits>
+          <div slot-scope="{ items, refine, insights }">
+            <div
+              v-for="item in items"
+              :key="item.objectID"
+            >
+              custom objectID: {{item.objectID}}
+              <button @click="insights('clickedObjectIDsAfterSearch', { eventName: 'Add to cart', objectIDs: [item.objectID] })">Add to cart</button>
+            </div>
+            <button
+              :disabled="isLastPage"
```

Comment:
This variable is not declared inside the `scope`.

Reply:
fixed here 3132a34